### PR TITLE
Improve Logging::Railtie activation warning messages

### LINKED ISSUE #1095: Can't run db:migrate after adding stackdriver gem
Repro:
1. Add stackdriver gem to the gem file of a project using App Engine Flex and Cloud SQL
2. run RAILS_ENV=production bundle exec rake db:migrate to migrate the database before deploying

Result:
Unable to initialize Google::Cloud::Logging with empty project_id

Expected:
To be able to migrate my cloud sql instance.

Notes
gcloud init and gcloud auth login had already been run. gcloud app deploy was working.

### PR DESCRIPTION
**Pull request by hxiong388**
Improve the Google::Cloud::Logging::Railtie activation warning messages. So they don't get confused as error message. See #1095 for example.

Also, the new Google::Cloud::Trace::Railtie warning messages changed the wording a little bit. So I'm also updating the warning message in Logging::Railtie to match that, so they are consistent.

[close #1095 ]

### CODE REVIEW

File: google-cloud-logging/lib/google/cloud/logging/rails.rb [hunk 1]
```diff
@@ -104,15 +104,15 @@ def self.use_logging? config
             Google::Cloud::Logging::Credentials.credentials_with_scope keyfile
           rescue Exception => e
             warn "Unable to initialize Google::Cloud::Logging due " \
-              "to authorization error: #{e.message}"
+              "to authorization error: #{e.message}\nFallback to default logger"
```
Comment (dazuma) [inline google-cloud-logging/lib/google/cloud/logging/rails.rb]: "Falling back to default logger."

File: google-cloud-logging/lib/google/cloud/logging/rails.rb [hunk 2]
```diff
@@ -104,15 +104,15 @@ def self.use_logging? config
...
             return false
           end
 
           project_id = gcp_config.logging.project_id || gcp_config.project_id ||
                        Google::Cloud::Logging::Project.default_project
           if project_id.to_s.empty?
-            warn "Unable to initialize Google::Cloud::Logging with empty " \
-              "project_id"
+            warn "Google::Cloud::Logging is not activated due to empty " \
+              "project_id; fallback to default logger"
```
Comment (dazuma) [inline google-cloud-logging/lib/google/cloud/logging/rails.rb]: "falling back to default logger."

**Comment by hxiong388** [comment]
@dazuma updated.

File: google-cloud-logging/lib/google/cloud/logging/rails.rb [hunk 3]
```diff
@@ -104,15 +104,15 @@ def self.use_logging? config
             Google::Cloud::Logging::Credentials.credentials_with_scope keyfile
           rescue Exception => e
             warn "Unable to initialize Google::Cloud::Logging due " \
-              "to authorization error: #{e.message}"
+              "to authorization error: #{e.message}\nFalling back to default logger"
```
Comment (dazuma) [inline google-cloud-logging/lib/google/cloud/logging/rails.rb]: Whoops looks like rubocop failed on this line due to line length.

**Pull request merged**